src/KeyLoader.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Acquia\Hmac;
+
+class KeyLoader implements KeyLoaderInterface
```

Comment:
Should have docblocks for class, any variables, and the constructor (see other classes for examples).